src/app/core/user-tools/user-tools.component.ts

```diff
@@ -1,14 +1,32 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Observable, Subject, Subscription } from 'rxjs';
+import { AuthService } from '../../auth/shared/services/auth.service';
+import { User } from '../../auth/shared/models/user.model';
 
 @Component({
   selector: 'app-user-tools',
   templateUrl: './user-tools.component.html',
   styleUrls: ['./user-tools.component.sass']
 })
-export class UserToolsComponent implements OnInit {
-  public userLogin: string;
+export class UserToolsComponent implements OnInit, OnDestroy {
+  public userEmail: string;
+  public subscription: Subscription;
+  public isAuth$: Observable<boolean>;
+
+  constructor(private auth: AuthService) {}
 
   ngOnInit() {
-    this.userLogin = 'User Login';
+    this.isAuth$ = this.auth.isAuthenticated();
+    this.subscription = this.auth
+      .getUserInfo()
+      .subscribe(user => (this.userEmail = user.email));
```

Comment:
Again, it can be done cleaner with smth like this:
`this.userEmail$ = this.auth.getUserInfo().pipe(map(user => user.email))`